Add advice on recording meetings (#975)
HSF/hsf.github.io: what_are_WGs.md

working group conveners can give input to the organising team. If there
will be a session dedicated to the activities of any working group then
conveners will be asked to co-organise it.

### Recording Meetings and Minutes

> In general recording *is appropriate* for HSF meetings, which are there to

Reviewer: ```suggestion
In general, recording *is appropriate* for HSF meetings, which are there to
```

> foster discussion and collaboration so *we do recommend it*. There are a lot of

Reviewer: ```suggestion
foster discussion and collaboration, so *we do recommend it*. There are a lot of
```

meetings and we have a worldwide community, so not everyone who is interested

> can join the meeting and a recording can help people to catch-up.

Reviewer: ```suggestion
can join a meeting and recordings can help people to catch-up.
```

> However, you need to make make it very clear with the speakers and to participants that:

Reviewer: ```suggestion
However, you need to make it very clear with the speakers and to participants that:
```

> - That the meeting is being recorded and will be made public;

Reviewer: ```suggestion
- The meeting is being recorded and will be made public (most likely on Indico and/or YouTube);
```

### Recording Meetings and Minutes

In general, recording *is appropriate* for HSF meetings, which are there to
foster discussion and collaboration, so *we do recommend it*. There are a lot of
meetings and we have a worldwide community, so not everyone who is interested
can join a meeting and recordings can help people to catch-up.

However, you need to make it very clear with the speakers and to participants that:

> - The meeting is being recorded and will be made public;

Reviewer 2: IMHO the "that" belongs to the beginning of the line (removing it from above). Just because of the following bullet point

Author: Yes, agreed - thanks.
